app/src/main/java/org/oppia/android/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -283,17 +306,13 @@ class StatePlayerRecyclerViewAssembler private constructor(
       }
     }
 
-    if (isTerminalState && playerFeatureSet.showCelebrationAtEndOfSession) {
-      maybeShowCelebrationForEndOfSession()
-    }
```

Comment:
Here & below: these seem like unrelated changes. Why are they needed?

Reply:
Probably this might be caused due to conflict errors.

Reply:
Please double check your work before sending it for review. You should make sure your code is what you intend to check in before asking others to check it.

Reply:
Thanks, I'll keep this in mind